supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc

```diff
@@ -78,6 +78,17 @@
 
 *See also*: xref:distinguished-name[distinguished name]
 
+[discrete]
+[[baseos-repository]]
+==== image:images/yes.png[yes] BaseOS repository (noun)
+*Description*: RHEL content in the _BaseOS repository_ provides the core set of the underlying OS functionality that provides the foundation for all installations.
```

Comment:
Since this seems pretty specific to RHEL, should probably reword to start with "In Red Hat Enterprise Linux,"

I see you did do that with a few other terms, so not sure if it's not specific to RHEL. But maybe do another quick look throughout the PR to start with that for all that are pretty specific to only RHEL?